go.mod

```diff
@@ -34,16 +34,16 @@ require (
 	golang.org/x/tools v0.1.11
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	helm.sh/helm/v3 v3.9.0
-	k8s.io/api v0.24.1
-	k8s.io/apiextensions-apiserver v0.24.1
-	k8s.io/apimachinery v0.24.1
+	k8s.io/api v0.24.2
+	k8s.io/apiextensions-apiserver v0.24.2
+	k8s.io/apimachinery v0.24.2
 	k8s.io/cli-runtime v0.24.1
-	k8s.io/client-go v0.24.1
+	k8s.io/client-go v0.24.2
 	k8s.io/kubectl v0.24.1
```

Comment:
helm-plugins didn't get bumped?

Reply:
@varshaprasad96 

I thought to do that in the follow up so we can get the fix: https://github.com/operator-framework/helm-operator-plugins/pull/191

Could that be?

Reply:
Yeah, sure 👍